src/scenes/home/team/executiveStaff.js

```diff
@@ -41,10 +34,10 @@ const CTO = {
 };
 
 const CDO = {
-  name: 'Kate Horner',
-  role: 'Acting CDO',
-  slackUsername: '@katehorner',
-  email: '',
+  name: 'Jennifer Weideman',
+  role: 'CDO',
+  slackUsername: '@Jenn',
```

Comment:
@jvillama Sorry for not clarifying it earlier - can Jenn's card list two roles? 
basicaly: `role: 'CCMO, CDO'` ?

Reply:
Sure, I can update that.